demisto_sdk/commands/content_graph/parsers/pack.py

```diff
@@ -246,6 +249,7 @@ def __init__(
         if path.name == PACK_METADATA_FILENAME:
             path = path.parent
         BaseContentParser.__init__(self, path)
+        self.structure_errors: List[StructureError] = self.validate_structure()
```

Comment:
isn't this inherited? 🤔

Reply:
No, This is in the line `self.structure_errors: List[StructureError] = Field(default_factory=list)`
but you have to implement it here with calling `self.validate_structure()` method